python/brim/src/zqext.go

```diff
@@ -61,7 +61,7 @@ func doZqlFileEval(inquery, inpath, informat, outpath, outformat string) (err er
 	}
 	defer rc.Close()
 
-	w, err := emitter.NewFile(outpath, zio.WriterOpts{
+	w, err := emitter.NewFile(context.Background(), outpath, zio.WriterOpts{
```

Comment:
Shouldn't this be TODO?
```suggestion
	w, err := emitter.NewFile(context.TODO(), outpath, zio.WriterOpts{
```

Reply:
I used `context.Background` since that's what's used below for the call to `driver.Run`.